Added edit button to comments of doc timeline

dweipert-3138720606 opened the pull request:
Like suggested here https://github.com/frappe/frappe/issues/2454

Seems like my less compiler lowers the hex color. duh.

Commit:
Added edit button to comments of doc timeline

DRogue1337 commented on frappe/public/js/frappe/form/footer/timeline.js:
```diff
@@ -128,7 +128,33 @@ frappe.ui.form.Timeline = Class.extend({
-			});
+			})
+            .on('click', '.edit', function() {
+                var is_editing = 'is-editing';
```
When on keydown enter, the comment is submitted, then shift+enter needs to be used to make a new line(But that's the case in many editors anyway). Maybe change the edit icon to a "tick" so it gets more obvious that you should click on it once editing is finished?

DRogue1337 replied:
I used the check-icon solution for now.
Removing the textarea border seemed somewhat out of place and leaving a missing hint that you can click in it for editing now. Could make it thinner though.

Commit:
Used $timeline_item where applicable
Changed pencil-icon to check-icon while editing
Added hover effect to edit icon
Made octicon-check green and slightly bigger

rmehta commented on frappe/public/js/frappe/form/footer/timeline_item.html:
```diff
@@ -17,6 +17,11 @@
 					{%= data.delete %}
 				</span>
 			</div>
+			<div class="pull-right edit-btn-container">
```
tabs?

DRogue1337 replied:
Will do

rmehta commented:
@DRogue1337 do you want to change spaces to tabs? we consistently use tabs in the project!

Commit:
Used tabs instead of spaces for indentation

rmehta committed:
Update utils.py

rmehta committed:
Update timeline.js

rmehta committed:
Merge branch 'develop' into develop
